data.md

```diff
@@ -0,0 +1,66 @@
+# OpenRitardi Datasets
+We welcome any sort of analysis on Italian train data. We would be happy to know about your data analysis ideas and even provide you with feedback and support in that. If you obtain some interesting findings or data visualizations, we would be glad to include them on OpenRitardi's [website](https://www.openritardi.it) and credit you accordingly. 
+
+## Introduction
+### What is OpenRitardi?
+OpenRitardi is the first visualization tool of train delays in Italy. You can see the performance of each station and train, and see where the railway service can be improved. The project showcases how open data allows accountability and improves quality of service. OpenRitardi is *open source project* published on [GitHub](https://github.com/giacomoorsi/OpenRitardi).
+
+### Data source
+Data is not collected by OpenRitardi directly, but it collected by [TrainStats](https://trainstats.altervista.org/), which is a service that downloads daily data from ViaggiaTreno, the Trenitalia's website that lists train delays. OpenRitardi's contributors have used and analyzed the data, and we are happy to provide you with some guidelines. 
+
+## Data processing guidelines
+### Obtaining data
+You can download datasets from [TrainStats's Mega folder](https://mega.nz/folder/jRImnIqD#XrqeWFZ3h12wS8O_U4APXQ). Choose the days that you want to download (a period of 3-month data is about 10GB). 
+### Data raw format
+TrainStats datasets contain: 
+
+1. an entry for everytime a train stops at a station in a specific day, and contains information on the train (train class, number, departure, destination), the station, the scheduled time, the actual time and the delay
+2. general information on the railway status for a specific day, like Trenitalia or RFI text announcements
+3. broad statistics on the number of trains circulated on a specific day
+
+For OpenRitardi usecases we are mainly interested by 1. 

```

Comment:
I would rather say that up till now we used 1. Using the announcements might be cool!